modules/index.js

```diff
@@ -0,0 +1,91 @@
+import { DateTime } from './luxon.js';
+import Book from './book.js';
+import Books from './books.js';
+import Library from './library.js';
+
+function displayBooks() {
```

Comment:
Great design @mrcbq 🤝

- [ ] As requested by the previous reviewer, kindly use the `arrow syntax` instead of the `function` keyword for all your functions.

Reply:
OK, Thanks for your advice.
It's done.